src/pages/home/ReportScreen.tsx

```diff
@@ -625,6 +626,18 @@ function ReportScreen({
         );
     }
 
+    Log.info(
```

Comment:
I think this would be too noisy to log in the server. Let's use `Log.client` instead?

```suggestion
    Log.client(
```

Reply:
Those will still be available in device logs that are posted to perf channel right?